conf/evolutions/default/2.sql

```diff
@@ -0,0 +1,224 @@
+-- Create schema
```

Comment:
You do not need to rewrite the rest of the tables to migrate and only have items that have been added or changed in Migration.

Reply:
Fixed.